src/components/ui/Callout/Callout.tsx

```
@@ -9,13 +9,16 @@ export type CalloutProps = {
     className?: string ;
     color?: string;
+    variant?: string;
+    size?: string;
     customRootClass?: string;
     props?: object[]
 }
 
 const COMPONENT_NAME = 'Callout';
-const Callout = ({ children, className = '', color, customRootClass, ...props }: CalloutProps) => {
-    return (<CalloutRoot customRootClass={customRootClass} className={clsx(className)} color={color ?? undefined} {...props}>
+const Callout = ({ children, className = '', color, variant='', size='', customRootClass, ...props }: CalloutProps) => {
```

ESLint

[error] Operator '=' must be spaced.

(space-infix-ops)